### ISSUE TITLE
Create an option to both accounts unfollow each other withouth blocking them.

### ISSUE THREAD

**Issue by flaviofrancisco** [bluesky-social/social-app#5512]
**Is your feature request related to a problem? Please describe.**

I noticed the some random accounts of adult content is follow my account and I would not like to have them follow me (OK those ones I can block), but sometimes I no longer interested in someones content and I don't think also that person should be following me, but I don't want to block them.

**Describe the solution you'd like**

- We could do it in different ways like: blocking and unblocking both accounts don't follow each other.
- A drop down button: "Make both ways unfollow"

**Describe alternatives you've considered**

It is anoying because the only way is blocking the account for now.

**Additional context**

Our account is like our home we should be more in control to also decide who should or not follow us.

**Comment by mary-ext** [comment]
Duplicate of https://github.com/bluesky-social/social-app/issues/1160
